### ISSUE TITLE
[Core] Hearthing/Porting Deaths/Damage

### ISSUE THREAD

**Issue by Jibb94** [WoWManiaUK/Redemption#1008]
Multiple times today while porting I experienced a problem where I would die after using the port or take a damage from a port.  

Upon further inspection I noticed that the game was counting it as falling damage.  I included a picture with this post to show this damage in the combat log.  

Another thing to note is the more times porting in one play session the more damage I would take from each subsequent port.  The death is only dependent on how much damage you take. 

I had a friend who experienced the same issues.  I also saw that there was a case opened on the matter already and it had been closed saying it was "unable to be replicated" https://github.com/WoWManiaUK/Redemption/issues/1006 but i can ensure this is a real problem.

![falling port problem](https://cloud.githubusercontent.com/assets/26099689/24346226/4a2fccae-1288-11e7-8720-fad2e86be1bf.png)

**Comment by Tybot204** [comment]
I too have experienced this within the last 24 hours of play. It did not occur every port, but has happened with hearthstones and mage portals.

**Comment by Plege** [comment]
So just as a theory. What if it actually calculates the difference from start to ending point. Last raid we did we went rs in the cellar of the tower ported many people from dala many died. Dala is up in the air so the difference from dala to the raid instance is quite high. But if you would stay on the same level or go to a higher point for example hearthing to dala from a lower point would be fine as you do bot fall down
